config/jobs/kubernetes-sigs/cluster-api/cluster-api-periodics-release-1-0.yaml

```diff
@@ -0,0 +1,129 @@
+periodics:
```

Comment:
Can you please also add a copy of `periodic-cluster-api-e2e-v1alpha3-v1beta1-upgrade-main` (from cluster-api-periodics-main.yaml)